packages/discord.js/src/structures/BaseGuildTextChannel.js

```diff
@@ -2,8 +2,8 @@
 
 const GuildChannel = require('./GuildChannel');
 const TextBasedChannel = require('./interfaces/TextBasedChannel');
+const ThreadManager = require('../managers/GuildTextThreadManager');
```

Comment:
Pardon me if this is wrong, but shouldn't the variable be renamed to `GuildTextThreadManager` as well?

```suggestion
const GuildTextThreadManager = require('../managers/GuildTextThreadManager');
```